project/plugins.sbt

```diff
@@ -2,3 +2,4 @@
 // remember to also update build.sbt
 addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-theme" % "0.10.3")
 addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.11")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
```

Comment:
@jrudolph Newline is missing, scalafmt adds new lines

Reply:
:+1: done